chain/jsonrpc/res/network_info.js

```diff
@@ -38,10 +38,23 @@ function computeTraffic(bytes_received, bytes_sent) {
     return "⬇ " + convertBps(bytes_received) + "<br>⬆ " + convertBps(bytes_sent);
 }
 
-function add_debug_port_link(peer_addr) {
+function add_debug_port_link(peer_network_addr) {
+    // Assume rpc port is always 3030 and network port is always 24567 for the first neard process on a machine
+    // Then, each subsequence neard process on the same machine will are strict increments of a fix constant from both ports
+    // peer_num should only be > 0 for nearup's localnet and 0 otherwise
+    // Reference to nearup's localnet port assumptions
+    // https://github.com/near/nearup/blob/0b9a7b60236f3164dd32677b6fa58c531a586200/nearuplib/localnet.py#L93-L94
```

Comment:
Please don't use nearup as reference.

Reply:
This issue mainly addresses localnet configurations and this issue does not exist in mainnet/testnet/mocknet (any network that runs 1 process per computer as it doesn't end up with clashes in port usage (3030 and 24567). 
What alternatives can I use as reference for localnet port configurations?

Reply:
Something like that: https://github.com/near/nearcore/blob/cc38c88bdcdabd243dffa753b71e0d06cc19f0e4/chain/network/src/tcp.rs#L186

The issue with `nearup` is that it's use is discouraged for anything other than betanet and localnet. Pagoda is going to stop using nearup for betanet. And recently `neard localnet` command received some care and the value of using nearup for localnet is even smaller.

Reply:
Will remove any mention of nearup or links to it in code